bootstrap/.gitignore

```diff
@@ -9,6 +9,7 @@ dist/*
 dts/*
 web/*
 !web/iframe-adapter.js
+!web/assets
```

Comment:
Feel free to correct me if I'm wrong! :

For plugin-side assets, these should be stored in /webClient/  which then get compiled into /web/. So that means translation file should exist in webClient, and just as with all compiled things, we don't include them into Git and thus web/assets should continue to get ignored because it will get generated

Reply:
Actually, let's keep this in web/assets. Bootstrap is one of those things that is unusual to build